src/core/VarReplay.jl

```diff
@@ -305,8 +301,8 @@ end
 #   vi.logp = vi.logp[end:end]
 # end
 
-# Get all indices of variables belonging to spl.selector or DEFAULT_SELECTOR
-getidcs(vi::VarInfo, ::SampleFromPrior) = filter(i -> DEFAULT_SELECTOR in vi.gids[i] , 1:length(vi.gids))
+# Get all indices of variables belonging to SampleFromPrior
```

Comment:
consider adding a comment here to indicate that: if `VarInfo.gid` is empty for a variable, then it's assumed to be assigned to a `SampleFromPrior` sampler.

Reply:
done.